static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-// import { onConsentChange, getConsentFor } from '@guardian/consent-management-platform';
+import { onConsentChange, getConsentFor } from '@guardian/consent-management-platform';
 import config from 'lib/config';
 import { loadScript } from '@guardian/libs';
 
```

Comment:
Not part of this PR, but if the definitions on line 6 and 7 are just to make linting happy, I'd personally suppress the errors rather than define them. It's a bit unclear otherwise why we have them.